src/app/OperationalDeviceProxy.h

```diff
@@ -273,6 +279,13 @@ class DLL_EXPORT OperationalDeviceProxy : public DeviceProxy,
      * This function will set new IP address, port and MRP retransmission intervals of the device.
      */
     void UpdateDeviceData(const Transport::PeerAddress & addr, const ReliableMessageProtocolConfig & config);
+
+    static void HandleBackoffTimer(System::Layer * aSystemLayer, void * aAppState);
+    void TryRecoverSession();
+    void RecoveringBackoff();
```

Comment:
This is purely editorial, but these method names make more sense to me:

```suggestion
    void TrySessionRecovery();
    void RecoveryBackoff();
```